src/app/features/ledger/flows/stacks-tx-signing/ledger-sign-stacks-tx-container.tsx

```diff
@@ -148,11 +150,9 @@ function LedgerSignStacksTxContainer() {
     hasUserSkippedBuggyAppWarning,
   };
 
+  const canClose = !awaitingDeviceConnection && canUserCancelAction;
```

Comment:
Any way we can avoid the repetition of this expression?

Reply:
Sure, I will refactor it 👍

Reply:
@kyranjamie: I did a refactor to improve this [here](https://github.com/leather-wallet/extension/pull/5353/commits/01dbb335f608c41467c26f1e46efc6813d146025). 

I added some more notes [here](https://github.com/leather-wallet/extension/pull/5353/commits/01dbb335f608c41467c26f1e46efc6813d146025#r1599656683)